packages/gui/src/types/TypographyInputs.tsx

```diff
@@ -0,0 +1,14 @@
+import { EditorProps } from '../types/editor'
+import { FontFam } from '../types/css'
```

Comment:
There a reason why this isn't `FontFamily`? Can always `import { FontFamily as FontFamilyType }` to avoid naming conflicts.